### ISSUE TITLE
Implement new design and links for navbar and footer

### ISSUE THREAD

**Issue by angela-tran** [cal-itp/calitp.org#93]
Depends on #91 and #92 so we can add links to those pages

The footer has a new link ("Back to top"). Both navbar and footer require some CSS changes.

## Tasks

- [ ] #98
- [ ] #99

**Comment by thekaveman** [comment]
Closing in favor of the more specific tickets linked above.
